docs/internals/deprecation.txt

```diff
@@ -26,6 +26,9 @@ details on these changes.
   compatibility layer which allows absolute URLs to be considered equal to
   relative ones when the path is identical will also be removed.
 
+* The ``GeoManager`` and ``GeoQuerySet`` classes will be removed, as all custom
```

Comment:
will be removed. (can chop the rest -- that explanation is for the release notes; timeline just lists things that are removed)